.travis.yml

```diff
@@ -12,6 +12,7 @@ jdk:
   - oraclejdk8
 before_install:
   - mkdir "${ANDROID_HOME}/licenses" || true
+  - echo -e "\nd56f5187479451eabf01fb78af6dfcb131a6481e" > "${ANDROID_HOME}/licenses/android-sdk-license"
```

Comment:
That doesn't seem to work either. Maybe the license hash is wrong or maybe the license name?

Reply:
yeah, I saw but I was pretty much out of ideas. It's pretty nasty to test too. I can just force push...